src/types/fungible-position.ts

```diff
@@ -0,0 +1,78 @@
+import { Links } from "./common";
+
+export interface FungiblePositionsResponse {
+  links: Links;
+  data: PositionData[];
+}
+
+export interface PositionData {
+  type: "positions";
+  id: string;
+  attributes: PositionAttributes;
+  relationships: PositionRelationships;
+}
+
+export interface PositionAttributes {
+  parent: null | string; // Could potentially be a string if there's a parent, but it's null here
+  protocol: null | string; // Protocol might be null or could be defined
+  name: string;
+  position_type: "wallet";
+  quantity: PositionQuantity;
+  value: number | null; // Value might be null
+  price: number;
+  // TODO
+  // changes: null | unknown;
+  fungible_info: FungibleInfo;
+  flags: PositionFlags;
+  updated_at: string; // ISO date string
+  updated_at_block: number;
+}
+
+export interface PositionQuantity {
+  int: bigint; // Large integer in string format
```

Comment:
Might be to transport strings here (for frontend client<>server data sharing).

Reply:
In theory (ignoring JS precision issues), should this be true ? `
(int / 10 ** decimals) === float === parseFloat(numeric)
`

Reply:
I prefer to keep the bigint values until they need to be displayed in the interface. 

However I don't know what PositionQuantity is here (never really looked at the values contained in there).

Reply:
Maybe float is best here. But we wont use this for a while so I will just make it a string again for now.